### PR TITLE
[RISC-V] Port Mono for RISC-V 64 Arch (3/3) IL Lowering & Outputting

### PR DESCRIPTION
**Pull request by Xinlong-Wu**
These patches initially ported the JIT of MonoVM to `RISC-V 64 imafdc` Arch, withing `lp64d` ABI.
So that the runtime can run and output `hello world` to command line by MonoVM under these patches

image

This patch lowering and output some IL instructions used during print "Hello World" based on the instructions of RISC-V64. In addition, it improves some logging info for debugging purposes.

ps. Only applying this patch may not compile successfully, splitting the patch is only for review purposes. more info please refer to https://github.com/dotnet/runtime/issues/36748

### CODE REVIEW

File: src/mono/mono/mini/mini-ops.h [hunk 1]
```diff
@@ -1793,3 +1793,32 @@ MINI_OP(OP_CVT_SI_FP_SCALAR, "convert_si_to_fp_scalar", XREG, XREG, NONE)
...
+MINI_OP(OP_RISCV_SLTI, "riscv_slti", IREG, IREG, NONE)
+MINI_OP(OP_RISCV_SLTIU, "riscv_sltiu", IREG, IREG, NONE)
+
+// used for cfg->r4fp == FALSE
+MINI_OP(OP_RISCV_SETFREG_R4,"riscv_setfreg_r4", FREG, FREG, NONE)
+#endif
+
+#if defined(TARGET_RISCV64)
+MINI_OP(OP_RISCV_ADDUW, "riscv_adduw", IREG, IREG, IREG)
+#endif
```
Comment (vargaz) [inline src/mono/mono/mini/mini-ops.h:1824]: Needs a newline.

**Comment by vargaz** [comment]
Looks ok, please fix the formatting to conform to the mono coding conventions, i.e. spaces before ( and { etc. Look for the other source files for examples.

**Review by vargaz — APPROVED** [review]

**Comment by Xinlong-Wu** [comment]
it may conflicts again when second pr has been marged, I will resolve it then.

**Comment by vargaz** [comment]
Thanks!

**Pull request merged**